utilities/common.py

```diff
@@ -124,15 +124,35 @@ def get_binning_fakes_mt(mt_cut=40):
 def get_dilepton_ptV_binning(fine=False):
     return [0, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 15, 17, 20, 23, 27, 32, 40, 54, 100] if not fine else range(60)
 
-def get_gen_axes(flow=False, dilepton_ptV_binning=None):
+def get_gen_axes(flow=False, dilepton_ptV_binning=None, inclusive=False):
     if dilepton_ptV_binning is None:
         dilepton_ptV_binning = get_dilepton_ptV_binning()
+
     gen_axes = {
         "ptVGen": hist.axis.Variable(dilepton_ptV_binning, name = "ptVGen", underflow=False, overflow=flow),
-        "absYVGen": hist.axis.Regular(10, 0, 2.5, name = "absYVGen", underflow=False, overflow=flow),  
+        "absYVGen": hist.axis.Regular(10, 0, 2.5, name = "absYVGen", underflow=False, overflow=flow)
     }
+    if inclusive:
+        binning = (*gen_axes["absYVGen"].edges[:-1], 5.)
+        gen_axes["absYVGen"] = hist.axis.Variable(binning, name="absYVGen", underflow=False, overflow=flow)
     return gen_axes
 
+def get_default_ptbins(analysis_mode, unfolding=False):
+    vals = [30,26.,56.] if analysis_mode[0] == "w" else [36,26.,62.]
+    if unfolding:
+        vals[0] -= 2
+        vals[1] += 2
+    return vals
+
+def get_default_etabins(analysis_mode=None):
```

Comment:
Argument not needed

Reply:
It's there for implementation consistency (in case there ever is an analysis dependence)